Jasmine/Common/Helpers/Random.swift

```diff
@@ -10,10 +10,21 @@ enum Random {
     ///   - toInclusive: end of range, inclusive
     /// - Returns: a random number in the range
     static func integer(from: Int = 0, toInclusive: Int) -> Int {
-        assert(toInclusive > from, "`toInclusive` must be larger than `from` number")
+        assert(toInclusive > from, "`toInclusive` must be larger than `from`")
         return from + Int(arc4random_uniform(UInt32(toInclusive - from + 1)))
     }
 
+    /// Generates a random integer
+    ///
+    /// - Parameters:
+    ///   - from: start of range, inclusive
+    ///   - toExclusive: end of range, exclusive
+    /// - Returns: a random number in the range
+    static func integer(from: Int = 0, toExclusive: Int) -> Int {
+        assert(toExclusive > from + 1, "`toExclusive` must be larger than `from` by at least 2")
```

Comment:
Java's `Random.nextInt` also accepts 1. I think it's more like - what if it just happens somehow that we want to randomize in a `1x1` grid? I think instead of it crashing weirdly we should just allow it.